src/components/Publish/Actions/index.module.css

```diff
@@ -8,3 +8,13 @@
 .actions button {
   margin: 0 calc(var(--spacer) / 2);
 }
+
+.infoIcon {
+  width: 0.85em;
+  height: 0.85em;
+  cursor: help;
```

Comment:
with the button having `cursor: pointer`, the disabled button having `cursor: no-allowed`, I am pretty sure this cursor is impossible to trigger for a user

Reply:
One option could be to do it by overriding the style that's already been added to disabled buttons. e.g. passing in a new class to the button like this:

```css
.infoButton {
  cursor: help !important;
  pointer-events: all !important;
}
```